assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+/* jshint unused: false */
```

Comment:
This file contained some unused vars that seemed to have meaning.  One example is the function on line 16:

```javascript
function errorPage(message) {
      var redirect = URI('error').query({
        "error_description": message
      }).toString();
      $location.url(redirect);
    }
```
jshint wanted it removed.